minnie-kenny.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/sh
```

Comment:
it would be nice if we could pull in a master version of this file but that's probably outside the scope for this ticket

Reply:
I think we want the current version to make sure all eng's new code not adding any secrtes